project/profiles/capacity4more/modules/c4m/mail/c4m_mail/messages/c4m_og_invite_users/group_invite_user_subject.txt

```diff
@@ -0,0 +1 @@
+[personal:subject]
```

Comment:
It can be done but I think it would get confusing because the template filenames are the same as the email identifier ($key). It's the same for all the email templates in the `c4m_mail > messages` folder.